### DISCUSSION TITLE
Showcase: Flume used on a Microsoft/Equinor project

### DISCUSSION THREAD

**Discussion by canxerian** [chrisjpatty/flume#211] [Show and tell]
Hey everyone,

I'm a software engineer at Microsoft and was involved in building a prototype, for which I chose to use Flume as I was so impressed with its features and configurability.

Here's a brief glimpse of the project featured at this year's Ignite conference: https://youtu.be/oo16phgcALQ?t=583

![image](https://user-images.githubusercontent.com/1138440/210016807-e9654b08-9eb3-4649-9e9f-04219060794d.png)

You'll notice there's been quite a few modifications - whilst we achieved much of it by overriding CSS styles, we eventually had to fork. I do want to merge these additions in to the main project (I have pending pull request for a small portion of that work https://github.com/chrisjpatty/flume/pull/188 😉).  

Feel free to ask questions and I'll try my best to answer 🙂.

Thank you @chrisjpatty and every contributor for creating and maintaining this awesome project!

**Comment by chrisjpatty** [comment]
Oh wow thanks for sharing that's very impressive! Yeah I would love to talk to you about merging your changes back in. There's a significant typescript rewrite in progress so that might complicate it but regardless I'd love to see what we can collaborate on.

**Reply by canxerian** [reply]
Thanks! I'm also looking forward on collaborating and making Flume even better! Good luck with the rewrite 🙂

**Comment by idling-mind** [comment]
Looks really cool!
